apps/generator/test/test-templates/react-template/package.json

```diff
@@ -3,28 +3,55 @@
     "version": "0.0.1",
     "description": "Simple react-based template that showcases different generator features that we can test in integration tests",
     "scripts": {
-        "test": "echo \"Error: no test specified\" && exit 1"
+      "test": "echo \"Error: no test specified\" && exit 1"
     },
     "generator": {
-        "renderer": "react",
-        "apiVersion": "v3",
-        "hooks": {
-            "@asyncapi/generator-hooks": "createAsyncapiFile" 
+      "renderer": "react",
+      "apiVersion": "v3",
+      "hooks": {
+        "@asyncapi/generator-hooks": "createAsyncapiFile"
+      },
+      "parameters": {
+        "version": {
+          "description": "Custom version to be used"
         },
-        "parameters": {
-            "version": {
-                "description": "Custom version to be used"
-            },
-            "mode": {
-                "description": "development or production"
-            },
-            "asyncapiFileDir": {
-                "description": "This template by default also outputs the AsyncAPI document that was passed as input. You can specify with this parameter what should be the location of this AsyncAPI document, relative to specified template output."
+        "mode": {
+          "description": "development or production"
+        },
+        "asyncapiFileDir": {
+          "description": "This template by default also outputs the AsyncAPI document that was passed as input. You can specify with this parameter what should be the location of this AsyncAPI document, relative to specified template output."
+        },
+        "singleFile": {
+          "description": "Allow to generate conditionalGeneration file only if singleFile is set to false"
+        },
+        "singleFolder": {
+          "description": "Allow to generate conditionalGeneration folder only if singleFolder is set to false"
+        }
+      },
+      "conditionalGeneration": {
```

Comment:
conditionalGeneraton is now an new feature for the old one, `subject` - after such refactoring it should also be tested, no need for both (file and folder), just one case

Reply:
Added one more test case for the subject key thanks @derberg